### DISCUSSION TITLE
VS2019/2022 项目正确配置

### DISCUSSION THREAD

**Discussion by jacksonjim** [netease-im/NIM_Duilib_Framework#414] [General]
平台工具集需要修改为 Visual Studio 2017 - Windows XP (v141_xp)
![VS2019_prj_config](https://user-images.githubusercontent.com/3296695/165717719-5b5e9ef0-c15d-432e-80aa-3561f93d6b9f.png)
方便正常编译所有项目
同时需要在VC++的语言问题修改符合模式为：否
![VS2019_prj_config2](https://user-images.githubusercontent.com/3296695/165718101-370d1005-cde7-4ddb-989f-43b8d34937aa.png)

**Comment by helloobaby** [comment]
搞了半天重定义，原来是这个符合模式的问题

**Reply by jacksonjim** [reply]
是的呀，慢慢研究下

**Comment by jacksonjim** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
